src/solver/optimisation/post_process_commands.cpp

```diff
@@ -112,53 +112,93 @@ void RemixHydroPostProcessCmd::execute(const optRuntimeData& opt_runtime_data)
                           hourInYear);
 }
 
+// ----------------------------------
```

Comment:
Here we separate 2 post processes : 
- the update of **MRG price** depending on the value of **ENS**, after the step of CSR. 
  Indeed, for certain hours, step of CSR possibly changed the **ENS**, and **MRG price** has to be changed subsequently, depending on the new value of **ENS**.
- Compute and set new variables (for **ELIA**'s need)